h/cli/commands/authclient.py

```diff
@@ -30,19 +30,19 @@ def add(ctx, name, authority, type_):
     """
     request = ctx.obj["bootstrap"]()
 
-    authclient = models.AuthClient(name=name, authority=authority)
+    authclient_ = models.AuthClient(name=name, authority=authority)
```

Comment:
This is a little cludgy to me. I think we could go with `auth_client` or just `client` as it's pretty clear which kind we have by context.

The other way to go is to call `def authclient` something like `authclient_command` or `authclient_group`